dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -150,18 +153,16 @@ public Response updateType(@PathParam("id") final String id, final String json,
 
 					contentType = capi.save(contentType);
 
+					final List<String> workflowsIds = form.getWorkflowsIds();
+					workflowHelper.saveSchemesByContentType(id, user, workflowsIds);
 					response = Response.ok(new ResponseEntityView(new JsonContentTypeTransformer(contentType).mapObject())).build();
 				}
 			}
-		} catch (DotStateException e) {
-
-			response = ExceptionMapperUtil.createResponse(null, "Content-type is not valid ("+ e.getMessage() +")");
-
 		} catch (NotFoundInDbException e) {
 
 			response = ExceptionMapperUtil.createResponse(e, Response.Status.NOT_FOUND);
 
-		} catch (DotDataException e) {
+		} catch ( DotStateException | DotDataException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)